.travis.yml

```diff
@@ -1,76 +1,41 @@
-    - INSTALL_CACHE=$HOME/install-cache
-    - BUSTED_ARGS="-o htest -v --exclude-tags=flaky,ipv6"
-    - PLUGIN_NAME=session
-    - KONG_TEST_PLUGINS=bundled,$PLUGIN_NAME
-    - KONG_PLUGINS=bundled,$PLUGIN_NAME
-
-  matrix:
-    - OPENRESTY=$OPENRESTY_BASE
-      CASSANDRA=$CASSANDRA_BASE
-    - OPENRESTY=$OPENRESTY_LATEST
-      CASSANDRA=$CASSANDRA_LATEST
-
-before_install:
-  - git clone https://$GITHUB_TOKEN:@github.com/Kong/kong-ci.git
-  - source kong-ci/setup_env.sh
-  - git clone -b next https://github.com/Kong/kong.git kong-ce
+  # for Enterprise images
+  - BINTRAY_USERNAME=kong-automation@kong
+  - BINTRAY_REPO=kong
+  - secure: SalrcDiyLF1AMXJm1ik8ltrumfnI0EGEKx+D2OEx3WLwtP4cH6FJbYT3qkyoJ0dN1MSf5LyUHir5ilkVF2uNRK9YTjYdKTLDWVpS1QRxmruj0aduenSar/YsizHJ5fcRtpqfgnvCDpRL0F2EkVt4MV24dWM71KZWN0JR3AgTTGhMSMyXnG6jQr7bVB8BErgM+6K0K1Eqv0hnqIYU8iJAWz62zk4EVZ6qfzPJa18E0+WYJ4qDssAqZI22qqYROz71BTYEXAi4mw9+TvaKIXyyZa1XO9bZQHD4nioi1IADHwWSq/ebOrcrg1a+2Ftqa+RRaB/Gwznt0AWnWWgcKH8b7Fx72R6TYvGYVhTLfPiWWLJ5wM342YKrqllHMiQ6HtbqC13rfTDKnrRlXPZ+p4CEAsGeVYA1UsYu/jL5wK0J9prN+Xj5HiXaOcJfCKrZ95vb6wmAY74iKayMy50X/dZvDXCJUszcYnkJkQbbYvvengxAMNe9XBdsMy9RE/lPuFW7XUQFmASn4kht2vcaQG8+9GweC8j9bn/7SNn87kytIS7kT2FTwc8Xtli5A9ErdPr8hkzWB9KGR5qgIWen7q/ha2CHKxLdg51LJvb24lHGunmzaDO4Jrr6l3yY5lLcarTZBrAMCF+Gdjyd/W5gJsLJQt73pK0tJNM0f+URoWssucI=
+  # for Nightly Enterprise images
+  - NIGHTLY_EE_USER=kong-automation@kong
+  - secure: XS7tRdo5TpZNmb0WhIQE7BzNqEtibU9SwqIGUFRlGwvZ3J+2/2q1QLxdQRUEmaCm1+PebA2SKkExH/gC3HfKE/Y3OeeoS4xLllxBFwzFL4xLul7QhcxxuvwYcaE8FDwVpXnNsJX0cpwcC9SrAin9V3CyezKMchWYQW+sGO7Q5HqeNnOpMO72c/JW31AqHxbdTKaILzjCM9XHj3IWw86WMNMV1MN0W/vJn18WsCRUwOq+iWLBDm9HNVIjtmmTrdOpck4Jkm58fWIXZZk5aDhIooie/kmnVN0jr7c8sUpa0sYCjwTzPmINhTFBnuqgpLS9cbJs2W26e1KsXhPmc4RL4G8lQLnNMRG4+zigkgV+HWAcss67JedvxTkxzUO5iM+MnSDg4pBHF2Ze6m96G+FvAemqKc54ISHuBTDTntkXkf4bMFZJ+X3OtNVfRaCv2Tu5AMYbRx85/UR4CWnWZvz8UZB7roBSEmjMaPptOzPE+4EOyZGmMwo92nHHROiqHi3aLtha0GP8FNBs4anlJvY1PAmi7crkS2VCQtZiOYtqzVytkU3w+PWW7RB37ewc1z5/OknP5e958tGRveoIAlr5o3jg1IDTi+kYFj4lzxxxv44x7jfyCZ8/j3Iz1UAql4o3LDBiRGciIUiq8XQAQiEwW9Ao3nqIuBy2ZveChk0WPXI=
 
 install:
-  - luarocks make
-  - cd kong-ce
-  - make dev
-  - createuser --createdb kong
-  - createdb -U kong kong_tests
-  - cd ../
-  - KONG_DATABASE=postgres KONG_PG_DATABASE=kong_tests kong-ce/bin/kong migrations bootstrap
-  - KONG_DATABASE=cassandra KONG_CASSANDRA_KEYSPACE=kong_tests kong-ce/bin/kong migrations bootstrap
-  - luarocks remove --force lua-resty-session
-  - luarocks install --force lua-resty-session 3.1
-  - cd kong-ce
+- git clone --single-branch https://github.com/Kong/kong-pongo ../kong-pongo
+- "../kong-pongo/pongo.sh up"
+- "../kong-pongo/pongo.sh build"
 
 script:
-  - cd .. && luacheck . && cd kong-ce
-  - bin/busted $BUSTED_ARGS ../spec
+- "../kong-pongo/pongo.sh lint"
+- "../kong-pongo/pongo.sh run"
 
-cache:
-  apt: true
-  pip: true
-  directories:
-    - $DOWNLOAD_CACHE
-    - $INSTALL_CACHE
-    - $HOME/.ccm/repository
+notifications:
```

Comment:
these setting seem chatty but to each their own

Reply:
only to verify the notifications are actually received